ts/pages/account/activity.tsx

```diff
@@ -145,17 +172,28 @@ export const AccountActivity: React.FC<ActivityProps> = () => {
                 <Heading
                     asElement="h1"
                     fontWeight="500"
-                    marginBottom="60px"
+                    marginBottom="30px"
                 >
                     Activity
                 </Heading>
+                {width > 600 ?
+                <ButtonWrapper>
+                    <Button
+                        isWithArrow={true}
+                        isAccentColor={true}
+                        shouldUseAnchorTag={true}
+                        onClick={() => {exportCSVFile(csvHeaders, delegatorHistory, 'staking_activity'); }}
```

Comment:
Would add some feedback in case the call fails, can be super simple
```
onClick={() => {
  try {
    exportCSVFile(csvHeaders, delegatorHistory, 'staking_activity'); 
  } catch (e) { 
     reportError(e);
     alert('Error exporting CSV. Try again later.');
  }}
}
```

Reply:
👍